packages/inertia-vue/src/directive.js

```diff
@@ -0,0 +1,36 @@
+import { Inertia, shouldIntercept } from '@inertiajs/inertia'
+
+export default {
+  bind(el, binding, vnode) {
+    el.addEventListener('click', event => {
+      if (!shouldIntercept(event)) {
+        return
+      }
+
+      event.preventDefault()
+
+      const method = binding.arg || 'get'
+      const options = (binding.value || {}).options || {}
+      const data = (binding.value || {}).data || (Object.keys(options).length === 0 ? binding.value : {})
+      const url = ((vnode.data || {}).attrs || {}).href || (binding.value || {}).href || (binding.value || {}).url
```

Comment:
One thing to add, I manually import this directive and this returns `undefined`. I had to change this to `el.href`

Reply:
Can you post an example of what you were trying exactly? I'll try to replicate it later, thanks!

Reply:
`el.href` needs to be added to line 15 or it won't work for this example

```
 <template>
    <v-navigation-drawer
      class="teal darken-3"
      dark
      permanent
    >
      <v-list>
        <v-list-item
          v-for="item in items"
          :key="item.title"
          link
          :href="item.route"
          v-inertia
        >
          <v-list-item-icon>
            <v-icon>{{ item.icon }}</v-icon>
          </v-list-item-icon>

          <v-list-item-content>
            <v-list-item-title>{{ item.title }}</v-list-item-title>
          </v-list-item-content>
          
        </v-list-item>
      </v-list>
    </v-navigation-drawer>
</template>
<script>
export default {
  data: () => ({
    items: [
      { title: "Dashboard", route: this.$route('dashboard.index'), icon: 'mdi-view-dashboard'},
      { title: "Organizations", route: this.$route('organizations.index'), icon: 'mdi-file-tree'},
    ]

  }),
};
</script>
```

`const url = ((vnode.data || {}).attrs || {}).href || (binding.value || {}).href || (binding.value || {}).url || el.href;`